citi-onboarding/iJunior-onepage
# Feature/frontend-banner

## What I did

- Thing I’ve done 
- Thing I’ve done
   - Detailing
   - Detailing

| Mockup | Real |
|--------|------|
| <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/122474373-7e3ffa00-cf99-11eb-9085-65022d3d60be.png"> | <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/122475336-e511e300-cf9a-11eb-8354-b311aaff8674.png"> |


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/frontend-banner`
   1. `git pull origin feature/frontend-banner`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `/client/src/views/banner`

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 4h36min

**Resolves:** #Frontend of the first section
